doc/guide/Guide_CLIMADA_conventions.ipynb

```diff
@@ -48,14 +54,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
    "source": [
     "## Class inheritance "
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
    "source": [
     "In Python, a [class can inherit from other classes](https://docs.python.org/3/tutorial/classes.html), which is a very useful mechanism in certain circumstances. However, it is wise to think about inheritance before implementing it. Very important, is that CLIMADA classes do not inherit from external library classes. For example, Exposure directly inherited from Geopandas. This caused problems in CLIMADA when the package Geopandas was updated.\n",
```

Comment:
I find this sentence difficult to read, suggestion:

Very important to note, that CLIMADA classes **do not** inherit from external library classes. For example, if `Exposure` class is directly inherited from the external package `Geopandas`, it may cause problems in CLIMADA if `Geopandas` is updated.